be/src/exec/hash_joiner.cpp

```diff
@@ -269,8 +268,6 @@ StatusOr<ChunkPtr> HashJoiner::_pull_probe_output_chunk(RuntimeState* state) {
             enter_eos_phase();
         }
 
-        RETURN_IF_ERROR(_filter_post_probe_output_chunk(chunk));
```

Comment:
Is moving this these two stmts required?

Reply:
I move it to HashJoinProber